apps/site/src/components/NavBar/NavBar.module.scss

```diff
@@ -3,73 +3,74 @@
 
 .nav {
 	width: 100%;
-	filter: drop-shadow(0px 2.5px 2px rgba(0, 0, 0, 0.4));
-
 	@include utils.media-breakpoint-up(sm) {
-		position: sticky;
+		position: fixed;
+		z-index: 1;
```

Comment:
Issue: parts of the navbar gets hidden underneath the Get Involved Section, so make sure that the navbar always has a high z-index. A solution would be to set the z-index to be 101 on line 18 and removing the other z-index's.